kernel/kernel-api/src/main/java/io/delta/kernel/types/StructField.java

```diff
@@ -50,6 +62,22 @@ public class StructField {
           FieldMetadata.builder().putBoolean(IS_METADATA_COLUMN_KEY, true).build(),
           Collections.emptyList());
 
+  public static StructField METADATA_ROW_ID_COLUMN =
+      new StructField(
+          METADATA_ROW_ID_COLUMN_NAME,
+          LongType.LONG,
+          false,
+          FieldMetadata.builder().putBoolean(IS_METADATA_COLUMN_KEY, true).build(),
+          Collections.emptyList());
+
+  public static StructField METADATA_ROW_COMMIT_VERSION_COLUMN =
+      new StructField(
+          METADATA_ROW_COMMIT_VERSION_COLUMN_NAME,
+          LongType.LONG,
+          false,
+          FieldMetadata.builder().putBoolean(IS_METADATA_COLUMN_KEY, true).build(),
+          Collections.emptyList());
```

Comment:
I would define these fields in RowTracking.scala instead of here, along with the field names `METADATA_ROW_ID_COLUMN_NAME` and `METADATA_ROW_COMMIT_VERSION_COLUMN_NAME`